tests/artifacts/test_artifacts.py

```diff
@@ -131,35 +135,227 @@ def test_generate_target_path():
             artifact, artifact_path, producer
         )
         print(f"\ntarget:   {target}\nexpected: {expected}")
+        assert target == expected
+
+
+def assets_path():
+    return pathlib.Path(__file__).absolute().parent / "assets"
+
+
+@pytest.mark.parametrize(
+    "artifact,expected_hash,expected_target_path,artifact_path,generate_target_path",
+    [
+        (
+            mlrun.artifacts.Artifact(key="some-artifact", body="asdasdasdasdas"),
+            "2fc62a05b53733eb876e50f74b8fe35c809f05c3",
```

Comment:
Not super critical but we talked about not putting the hashes in the parameters but rather calculating it using the hash functions directly. Consider doing this.

Reply:
so I changed part of them, reason I left some is to make sure we will fail in case that for some reason the hash output will change and we will be able to catch it.